Frameworks/CoreText/DWriteWrapper_CTFont.mm

```diff
@@ -225,11 +225,14 @@ static void __FilterFontsByTraits(const std::vector<ComPtr<IDWriteFont>>& fonts,
 static HRESULT __FilterFontsByStyleName(const std::vector<ComPtr<IDWriteFont>>& fonts,
                                         CFStringRef styleName,
                                         std::vector<ComPtr<IDWriteFont>>& matchingFonts) {
+    CFIndex styleLength = CFStringGetLength(styleName);
     for (auto& font : fonts) {
         ComPtr<IDWriteLocalizedStrings> dwriteFontName;
         RETURN_IF_FAILED(font->GetFaceNames(&dwriteFontName));
         CFStringRef name = _CFStringFromLocalizedString(dwriteFontName.Get());
-        if (CFStringCompare(name, styleName, kCFCompareCaseInsensitive) == kCFCompareEqualTo) {
+        CFIndex fontStyleLength = CFStringGetLength(name);
```

Comment:
nit: I'd just make a helper for the compare. You'd have to do more GetLengths but that shouldn't matter too much.